treeherder/settings/base.py

```diff
@@ -341,6 +341,11 @@
 # like ftp.mozilla.org or hg.mozilla.org
 TREEHERDER_REQUESTS_TIMEOUT = 30
 
+# the max size of a posted request to treeherder client.
+# If TreeherderCollections are larger, they will be chunked
+# to this size.
+TREEHERDER_CLIENT_CHUNK_SIZE = 50
```

Comment:
Yeah, fair point.  I went ahead and added specific settings values for ingestion of each pending, running, builds4h.  It feels like the right thing to have those in a settings file.  But that's more specific to the particular function.  Not just *all* of th client.  How does that sound?